src/screens/ProfileScreen/elements/BottomSheetRealname.js

```diff
@@ -35,55 +29,51 @@ const BottomSheetRealname = React.forwardRef((props, ref) => (
         style={styles.button}
         textStyling={styles.textStyling}
         onPress={() => props.handleSave()}>
-        {props.isChangeRealName ? (
-          <ActivityIndicator size="small" color="#0000ff" />
-        ) : (
-          'Save'
-        )}
+        {props.isChangeRealName ? <ActivityIndicator size="small" color="#0000ff" /> : 'Save'}
       </Button>
     </View>
   </BottomSheet>
 ));
 
 const styles = StyleSheet.create({
   containerBottomSheet: {
-    flexDirection: 'column',
+    flexDirection: 'column'
   },
   textYourName: {
     fontFamily: fonts.inter[800],
     fontWeight: 'bold',
     fontSize: 24,
-    color: colors.black,
-    marginBottom: 16,
+    color: COLORS.black,
+    marginBottom: 16
   },
   inputYourName: {
     height: 52,
-    backgroundColor: colors.wildSand,
-    color: colors.black,
+    backgroundColor: COLORS.wildSand,
+    color: COLORS.black,
     fontFamily: fonts.inter[400],
-    fontSize: 18,
+    fontSize: 18
   },
   descriptionYourname: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: colors.gray,
-    lineHeight: 24,
+    color: COLORS.blackgrey,
+    lineHeight: 24
   },
   errorText: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
     lineHeight: 24,
-    color: colors.red,
+    color: COLORS.redalert
   },
 
   button: {
     marginTop: 50,
-    backgroundColor: colors.bondi_blue,
+    backgroundColor: COLORS.anon_primary
   },
   textStyling: {
     fontFamily: fonts.inter[600],
     fontSize: 18,
-    color: colors.white,
-  },
+    color: COLORS.white
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles have been updated to use the new `COLORS` object for color values. This includes changes in text color, background color, and button color. These changes are consistent with the PR's goal of standardizing color usage across the application.
```diff
- color: colors.black,
+ color: COLORS.black,

- backgroundColor: colors.wildSand,
+ backgroundColor: COLORS.wildSand,

- color: colors.black,
+ color: COLORS.black,

- color: colors.gray,
+ color: COLORS.blackgrey,

- color: colors.red,
+ color: COLORS.redalert,

- backgroundColor: colors.bondi_blue,
+ backgroundColor: COLORS.anon_primary,

- color: colors.white,
+ color: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->